kernel/comps/systree/src/attr.rs

```diff
@@ -1,9 +1,12 @@
 // SPDX-License-Identifier: MPL-2.0
 
-use alloc::collections::BTreeMap;
+#![expect(clippy::type_complexity)]
```

Comment:
Please avoid adding such module-level lint attributes. Instead, you can choose to:

1. Add the lint attribute to each individual usage;
2. Use a type alias or trait alias.